Stackup-Rwanda/knights-bn-frontend
# #170947612 Approvals Table

#### What does this PR do?
- allow a user(administrator) to see a list of requests that require his/her approval on the Approvals page.
#### Description of Task to be completed?
- create approvals table.
- test the endpoint.
#### How should this be manually tested?
- run `git clone https://github.com/Stackup-Rwanda/knights-bn-frontend.git` to clone the repo.
- run `cd knights-bn-backend` in your terminal to enter into the repository.
- run `git fetch ` command to download commits, files, and refs from a remote repository into your local repo.
- run `git ft-approvals-table-170947612` to switch to this branch.
- run `npm install` to install dependencies.
- run `npm run dev-start` to start the application.
  you will see the login page then login with your administrator email or password or log in using your social account `Google & Facebook` 
- use `http://localhost:8080/pending` to access your approvals table.
#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

[#170947612](https://www.pivotaltracker.com/story/show/170947612)
#### Screenshots (if appropriate)

       approvals table

<img width="1440" alt="Screen Shot 2020-05-26 at 08 28 26" src="https://user-images.githubusercontent.com/50768983/82867449-32a05580-9f2b-11ea-956a-55744a5a9bf5.png">


         non-administrators not allowed to access the page.


<img width="1440" alt="Screen Shot 2020-05-26 at 08 27 57" src="https://user-images.githubusercontent.com/50768983/82867523-4fd52400-9f2b-11ea-96a6-0dc64b21d6ea.png">

#### Questions:
